spec/unit/action/install_chef/base_spec.rb

```diff
@@ -194,4 +162,69 @@
       end
     end
   end
+
+  context "#check_minimum_chef_version!" do
+    let(:target) { install.target_host }
+    context "when chef is not already installed on target" do
+      before do
+        expect(install.target_host).to receive(:installed_chef_version).
```

Comment:
Because the `check_minimum_chef_version` method takes in a `target` parameter and operates on it, I suspect all the uses of `install.target_host` in the `"#check_minimum_chef_version!"` spec context should be `target`.